src/libraries/System.Private.CoreLib/src/System/Number.NumberBuffer.cs

```diff
@@ -28,17 +28,16 @@ internal static partial class Number
             public NumberBufferKind Kind;
             public Span<byte> Digits;
 
-            public NumberBuffer(NumberBufferKind kind, byte* digits, int digitsLength)
+            public NumberBuffer(NumberBufferKind kind, Span<byte> digits)
             {
                 Debug.Assert(digits != null);
-                Debug.Assert(digitsLength > 0);
```

Comment:
nit: I think this assert is still valid, we still aren't expecting a zero length span.

Reply:
Done